### PR TITLE
Release v0.7.0

### PR DESCRIPTION
**Pull request by ayberk**
Release new version 0.7.0.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -46,6 +47,7 @@ Following sections are Kubernetes specific. If you are Kubernetes user, use foll
 | AWS EBS CSI Driver \ Kubernetes Version| v1.12 | v1.13 | v1.14 | v1.15 | v1.16 |
```
Comment (ayberk) [inline docs/README.md:47]: I left this as is, but I can update the supported versions if needed.

File: CHANGELOG-0.x.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
+# v0.7.0
+
+## Notable changes
+### New features
+* Add arm support ([#527](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/527), [@leakingtapan](https://github.com/leakingtapan))
+* Add EBS IO2 support ([#558](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/558), [@ayberk](https://github.com/ayberk))
+* Create volumes in outpost for outpost instances ([#561](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/561), [@ayberk](https://github.com/ayberk))
+
+### Improvements
+* Change the base image to debian-base ([#551](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/551), [@vpnachev](https://github.com/vpnachev))
```
Comment (wongma7) [inline CHANGELOG-0.x.md:10]: have to postpone this one, can remove the line

File: docs/README.md (hunk 1 above)
Comment (wongma7) [reply docs/README.md:47]: columns for 1.17 and 1.18 wont hurt, can update in other pr

**Pull request merged**